### ISSUE TITLE
Sass can't find bourbon / neat / normalize when used with Brunch

### ISSUE THREAD

**Issue by svoynow** [18F/web-design-standards#1722]
I'm trying to install this using npm into a Phoenix (http://www.phoenixframework.org/) application and running into issues with the Sass.
Phoenix uses http://brunch.io/ as its front end build tool, and it has a plugin for dealing with Sass
(https://github.com/brunch/sass-brunch)

The basic plumbing seems to be working. It is clearly parsing my @import directive to bring in the `uswds.scss`, but then it can't resolve the `@import lib\bourbon` line in `uswds.css`

`Error: File to import not found or unreadable: lib/bourbon`

I can provide a reproduction case, but it will involve a lot of stage setting (Phoenix project setup etc.)

Basically I'm just trying to find out how that name is *supposed* to resolve, since that path doesn't obviously correspond to anything in `uswds` module. I just want to make sure it's not a build problem or something on your end before I get deeper into the weeds with Brunch.

Thanks!

**Comment by juliaelman** [comment]
@svoynow thanks for your question and interest in using the Standards. 

Without being able to poke around in the code base, my initial guess is that `sass-brunch` expects the `@import lib/bourbon` path to resolve at the root level of the dir instead of the root level of the `uswds` node module folder. I would take a closer look at how pathing is compiled.

Going to close this issue, but please stay in touch and let us know how everything worked out!
